Front-end/src/components/sub-component/Comment.jsx

```diff
@@ -38,7 +38,10 @@ const Comments = (props) => {
     <Comment>
       <CommentAvatar src="https://react.semantic-ui.com/images/avatar/small/matt.jpg" />
       <CommentContent>
-        <CommentAuthor as="a" onClick={() => window.location.href=`/profile-${comment.id}`}>
+        <CommentAuthor
+          as="a"
```

Comment:
Give it a better name instead of "a" ? i.e. commentAuthorStyle

Reply:
I was following the documentation from semantic-ui-react so I had it as "a". I will test the change tomorrow to make sure I do not break my code with the changes.

Reply:
<img width="535" alt="Screenshot 2024-07-11 at 10 43 41 AM" src="https://github.com/Mar1789/Mentorship-Capstone-Project/assets/116681148/204abd80-17aa-495e-bc4c-2b64920493a8">

Warning I get if I change it.